central/graphql/resolvers/node_vulnerabilities.go

```diff
@@ -47,8 +52,7 @@ func (resolver *Resolver) NodeVulnerability(ctx context.Context, args IDQuery) (
 	if !features.PostgresDatastore.Enabled() {
 		return resolver.nodeVulnerabilityV2(ctx, args)
 	}
-	// TODO : Add postgres support
-	return nil, errors.New("Resolver NodeVulnerability does not support postgres yet")
+	return resolver.nodeCveV2(ctx, args)
```

Comment:
This indirection makes it readable. We do not expect logic in `nodeCveV2` to be accessible from anywhere else except this resolver. Therefore, no need to add a func.

Reply:
Yes the indirection would make it readable from outside. I had abstracted that code out to avoid function from getting too long.

Reply:
sorry...I really meant `less readable`. 😬